api-inference-community/tests/test_dockers.py

```diff
@@ -160,8 +160,13 @@ def test_timm(self):
         self.framework_invalid_test("timm")
```

Comment:
Not needed, but could you add a forked version of https://github.com/huggingface/huggingface_hub/blob/main/.github/workflows/python-api-allennlp.yaml that is modified for `keras`? We'll merge them in the future but I haven't gotten to investigating it.